src/axios-fetch-adapter.ts

```diff
@@ -0,0 +1,133 @@
+// @ts-nocheck
```

Comment:
This is a copy paste from a package called `axios-fetch-adapter`. I needed to fix some issues there so it was easier to just copy it. in the future we can probably use a fork.

Reply:
So this is a copy paste + the fixes?